src/components/Habit/Habit.js

```diff
@@ -1,24 +1,83 @@
 import React from 'react'
+import IconDelete from '../../components/assets/icons/IconDelete.svg'
 import styled from 'styled-components'
+import imagePathCircle from '../assets/icons/circle.svg'
+import imagePathCheckedCircle from '../assets/icons/checked-circle.svg'
 
-export default function Habit({ name, completed, id, onItemClick }) {
+export default function Habit({
+  category,
+  habits,
+  completed,
+  id,
+  onRemove,
+  onItemClick,
+  color,
+}) {
   return (
-    <HabitItem className="Habit">
-      {name}
-      <input
-        onChange={() => onItemClick(id)}
-        checked={completed}
-        type="checkbox"
-      />
-    </HabitItem>
+    <>
+      <Container style={{ backgroundColor: color }}>
```

Comment:
I would use style components here as well instead of the style attribute

Reply:
This belongs to the colorpicker und doesn't work with styled-components, unfortunately :(